apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/create-a-component.md

```diff
@@ -0,0 +1,290 @@
+
+1. From the app context flyout, select **Open App** to launch XM Cloud environment apps dashboard
+
+1. At the very top switch to **Tools**
+
+1. In the **Tools** page, select the **Components** card to launch the Component builder. You can see that there are already some components available.
+
+   `INSERT SCREENSHOT OF Components listing`
+
+1. On the top right, click **Add collection** to create a new collection.
+
+1. Name this collection: **“Basic Components”**
+
+1. Click the **Save** button to complete the collection creation.
+
+   `INSERT SCREENSHOT OF Basic Components collection`
+
+## Create the component
+
+Within this new collection, you will now create the new Component called “Text Teaser”.
+
+1. In the "Basic Components" collection, click the "Add component" button to launch the component creation form.
+
+1. Specify the **Name** value as **"Text Teaser"**.
+
+   <Alert status="info">
+      <AlertIcon />
+      You can leave the other fields blank. The **Description** field is optional and the **Collection** field will default to the currently selected Collection so it will be stored in Basic Components as desired.
+   </Alert>
+
+   `INSERT SCREENSHOT OF completed form`
+
+1. Click on the **Save** button to complete creating the component definition. A message will display telling you that the component was created.
+
+1. Click on the **Click to start editing** link that is displayed at the top of the Component form. This will launch the canvas (or 'Section') view of the Text Teaser component where you can start adding elements in the builder.
+
+   `INSERT SCREENSHOT OF Canvas`
+
+### Create the column layout for the Text Teaser component
+
+Before we begin adding content, we want to ensure we can provide a two-column layout that matches the requirements that were given at the beginning. You will now learn how to create Card elements that can be used to hold other elements. For more details on creating layouts, you can take a look at our documentation on [Working with a component layout](https://doc.sitecore.com/xmc/en/users/xm-cloud/work-with-a-component-layout.html)
+
+1. Click on the section/canvas. You will see a grid displayed by default.
+
+   <Alert status="info">
+      <AlertIcon />
+      This can be adjusted on the pane at the right side of the screen. For now leave it as it is.
+   </Alert>
+
+   `INSERT SCREENSHOT of editor with Layout&Alignment section highlighted with a red box`
+
+1. Click and drag your mouse across the canvas to create a left column for your Text Teaser.
+
+   <Alert status="info">
+      <AlertIcon />
+      By default, XM Cloud will create a **Block** element. You will want to change this to a **Card** in the following steps.
+   </Alert>
+
+   `INSERT SCREENSHOT of Canvas with drawn Block area`
+
+1. Click on the newly created left column Block area to load the context bar above the area.
+
+1. Click on the element type **Block** in the context bar to change the element type.
```

Comment:
Need to very this if really something called Block is clicked.

Reply:
Let me know! I added this based on my going through the steps in the UI, and I noticed that the context bar shows the name of the current element type. so if it's a Card, it shows Card, Block it shows Block, etc. In order to change it I had to click on the name of the current type to change it to another element type.

I wasn't entirely sure why it needed to be a Card? And we don't explain that in the tutorial either. Why is the default Block not good enough?